tests/integration/targets/vmware_cluster_drs/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
     password: "{{ vcenter_password }}"
     datacenter_name: "{{ dc1 }}"
     cluster_name: test_cluster_drs
-    enable_drs: false
```

Comment:
Can you also add a test that use `enable_drs` key on purpose, just to be sure we don't break existing users.

Reply:
@goneri 
> Can you also add a test that use `enable_drs` key on purpose, just to be sure we don't break existing users.

I've defined `enable_drs` as an alias of `enable`. It's the job of ansible itself that aliases work.

As far as I understand, it's even considered bad practice to check that the framework you're based on does it's job.

Thoughts?